test/client_service_base.test.js

```diff
@@ -89,7 +89,7 @@ describe('Class ClientServiceBase', () => {
         });
   });
   describe('The method', () => {
-    afterEach(function() {
+    afterEach(function () {
```

Comment:
Do we need the space ? It looks weird.

Reply:
Sorry it was the formatter used, i will try to change the setting of my intellij